src/OrbitGl/CoreMath.h

```diff
@@ -21,20 +21,34 @@ using Vec4i = gte::Vector4<int>;
 
 using Color = gte::Vector4<unsigned char>;
 
+// TODO(b/229089446): Test these methods.
 [[nodiscard]] inline Vec2 Vec3ToVec2(const Vec3& v) { return {v[0], v[1]}; }
 [[nodiscard]] inline Vec3 Vec2ToVec3(Vec2 vertex, float z = 0) { return {vertex[0], vertex[1], z}; }
 
-[[nodiscard]] inline bool IsInBetween(float point, float min, float max) {
-  return point >= min && point <= max;
+namespace orbit_gl {
+
+struct ClosedInterval {
+  static ClosedInterval FromValues(float value_1, float value_2) {
+    return {std::min(value_1, value_2), std::max(value_1, value_2)};
+  }
```

Comment:
I dont understand why this "complicated" method. AFAIK, wherever you use this method, it is already know which value is min (always value_1) and which is max (always value_2). 

I would do a simple explicit constructor.

Reply:
Discussed offline